src/apiCalls.js

```diff
@@ -0,0 +1,17 @@
+const getAllMovies = () => {
+    return fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies`)
+        .then(response => response.json())
+}
+
+const getMovieDetails = (id) => {
+    return fetch (`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}`)
+        .then(response => response.json())
+}
+
+const getMovieTrailer = (id) => {
+    return  fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}/videos`) 
+        .then(response => response.json())
+}
+
+
```

Comment:
This is definitely more legible and easy to read. Lots of developer empathy. I love it. My only feedback would be that we could use implicit returns to make it more concise.